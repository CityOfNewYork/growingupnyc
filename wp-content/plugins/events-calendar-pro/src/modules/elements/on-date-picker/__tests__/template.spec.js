import React from 'react';
import renderer from 'react-test-renderer';
import OnDatePicker from '../template';

describe( 'OnDatePicker', () => {
	test( 'should render component', () => {
		const component = renderer.create(
			<OnDatePicker
				className="test-class"
				date="September 21, 2019"
				onDateChange={ jest.fn() }
			/>
		);
		expect( component.toJSON() ).toMatchSnapshot();
	} );
} );
