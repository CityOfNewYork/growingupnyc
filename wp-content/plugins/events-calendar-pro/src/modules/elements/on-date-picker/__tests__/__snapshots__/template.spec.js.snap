// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OnDatePicker should render component 1`] = `
<div
  className="tribe-editor__events-pro__row tribe-editor__labeled-row tribe-editor__on-date-picker test-class"
>
  <div
    className="tribe-editor__events-pro__label tribe-editor__labeled-row__label"
  >
    <span
      className="tribe-editor__events-pro__label-text"
    >
      On
    </span>
  </div>
  <div
    className="tribe-editor__labeled-row__content"
  >
    <div
      className="tribe-editor__day-picker-input DayPickerInput"
    >
      <input
        onBlur={[Function]}
        onChange={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        placeholder="YYYY-M-D"
        value="September 21, 2019"
      />
    </div>
  </div>
</div>
`;
